nautobot/dcim/tables/template_code.py

```diff
@@ -62,9 +62,51 @@
 """
 
 INTERFACE_IPADDRESSES = """
-{% for ip in record.ip_addresses.all %}
-    <a href="{{ ip.get_absolute_url }}">{{ ip }}</a><br />
-{% endfor %}
+{% if record.interface.ip_addresses %}
+    {% for ip in record.interface.ip_addresses.all %}
+        <a href="{{ ip.get_absolute_url }}">{{ ip }}</a><br />
+    {% endfor %}
+{% else %}
+    --
+{% endif %}
+"""
+
+INTERFACE_REDUNDANCY_GROUP_INTERFACES = """
+{% if record.interfaces %}
```

Comment:
Same question here.

Reply:
thanks - I removed the conditionals from the new template code snippets.